lib/utils/analytics/generatePayload.js

```diff
@@ -85,5 +93,11 @@ module.exports = async (serverless) => {
     npmDependencies,
     triggeredDeprecations: Array.from(triggeredDeprecations),
     versions,
+    createdAt: Date.now(),
+    ciName: ci.name,
+    frameworkId: userConfig.get('frameworkId'),
```

Comment:
I would probably name it `installationId` (at least `frameworkId` gives wrong hint what it's about)

Although I agree, that switching to very different naming nomenclature also comes with downsides.

Reply:
I think in this particular situation `installationId` is a better name :+1:

Reply:
Do you think that in this case we should nest `installationId` and `installationType` under a single `installation` namespace?

Reply:
Good question, this  actually signaled to me that `installationId` is also not a good name (and I would definitely not mix it with `installationType`) :)

It's more about _system user_. As user may have multiple framework installations (global, local etc), and this installation can be updated, removed, installed again, and this id will stay same.

So what do you think best name for this property would be? Maybe `localUserId` or `systemUserId` ?

Reply:
good point - as for the name, this one is quite hard - `localUserId` sounds reasonable, I was also thinking about `localFrameworkId` or just sticking to `frameworkId` - which one do you think is the best? Btw - I would still keep `frameworkId` as it is in `.serverlessrc` file I think

Reply:
How about `frameworkLocalUserId` (it contains `framework` and `id` tokens, just specifies it a bit more, so it reflects data real meaning) ?

Reply:
> Do you agree on still keeping the frameworkId property in .serverlessrc file?

Yes, definitely. I would not touch it (at least not at this point)

Reply:
changed :+1: